.github/workflows/terraform.yml

```
@@ -0,0 +1,99 @@
+
+      - name: Wait and Destroy
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        run: |
+          sleep 300
+          terraform destroy -auto-approve
+
+      - name: Cleanup Terraform lock in S3 on cancellation
+        if: cancelled()
+        run: |
+          aws s3 rm s3://viet-terraform/vpc-lab/terraform.tfstate.tflock
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)